### DISCUSSION TITLE
How to disable gathering of language id? (WIX0204 ICE03 Invalid Language Id)

### DISCUSSION THREAD

**Discussion by Korzhukov** [wixtoolset/issues#7992] [Questions]
I am creating an MSI package for a WinUI3 (C++/WinRT) app. The app's release folder is full of **en-GB**, **ja-JP**, and similar subfolders with MUI files.

In WIX (version 4), there is no sign of a language ID after harvesting. However, the build fails with `WIX0204 ICE03 Invalid Language Id` for a few files located in folders like **mi-NZ** and **ug-CN**.

If I suppress the error and get the MSI file, Orca shows that all MUI and almost all DLL files have language IDs.

How can I add all these MUI and DLL files as language-neutral?

WIX3 had `<SuppressFileHashAndInfo>`, which, according to the manual, also suppressed the gathering of language information. Is this what I am looking for? If yes, it does not work in 4.0.4.

**Comment by jjensen4** [comment]
If you do not need the multi-language files, then adding the following line to the .Net Project resolved the Invalid Language Id errors in WIX:
`<SatelliteResourceLanguages>en</SatelliteResourceLanguages>`

**Reply by Korzhukov** [reply]
This only works for .Net projects, mine is C++
